source/_integrations/home_connect.markdown

```diff
@@ -34,7 +34,7 @@ Note that it depends on the appliance and on API permissions which of the featur
 
 - Application ID: Home Assistant (or whatever name makes sense to you)
 - OAuth Flow: Authorization Code Grant Flow
-- Redirect URI  : `<YOUR_HOME_ASSISTANT_URL>/auth/external/callback`
+- Redirect URI  : `<YOUR_INTERNAL_HOME_ASSISTANT_URL>/auth/external/callback` # Use same the address defined as `internal_url` in your configuration settings. Example `http://192.168.1.24:8123/auth/external/callback`
```

Comment:
I think this doesn't make this clearer. As one does not have to define an internal URL in the settings. So what if I didn't set it?

Reply:
Ok, got your point on the config part.

Then just mention that it needs to be the local IP-address URL? I personally fell into that "trap" today trying to use my nabucasa URL.

Reply:
More confusion on the subject can be found here: https://community.home-assistant.io/t/0-110-home-connect-not-working/197602/13

Reply:
It doesn't have to be an IP-Address... `homeassistant.local` is a perfectly fine host to use in an internal URL. Or in my case, it is even a subdomain of my personal main domain that resolves locally.

Reply:
Ok, updated. Please check :)